io.qt.qtwebengine.BaseApp.json

```diff
@@ -37,6 +37,18 @@
                 "ln -s /app/lib/*/*.so* /app/lib"
             ],
             "modules": [
+                {
+                    "name": "icu67",
+                    "subdir": "source",
+                    "no-autogen": true,
```

Comment:
`"no-autogen": true,` is actually unneeded, but I'll wait for the current CI to end before pushing a change.